.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: ruby
 cache: bundler
 
+before_install:
+  - gem update bundler
```

Comment:
Not sure why I was [getting this error](https://github.com/bundler/bundler/issues/3558), but it seems like bumping bundler helped.